R/utils-shared.R

```diff
@@ -35,6 +35,70 @@ startRender <- function(data, output, author, reportName){
 
 ############ used in dvhydrograph-data, fiveyeargwsum-data, uvhydrograph-data ############ 
 
+getGroundWaterLevels<- function(ts, ...){
+  y <- as.numeric(ts$gwLevel[['groundWaterLevel']])
+  x <- ts$gwLevel[['dateString']]
+  time = as.POSIXct(strptime(x, "%Y%m%d"))
```

Comment:
oh wait, looks like that's how it has been...better to rename it if you've got the time